docs/examples/rails-api/ingress-external.yaml

```diff
@@ -0,0 +1,37 @@
+################################################################################################################
+# How this works:
+# The serviceName identifies the kubernetes services that should be associated.
+# Deployment Lifecycle should always be:  APP SERVICE --> Update this ingress with the new property
+# This ingress rule is used for any rails/node/java app that has a /health endpoint as the healthcheck
+################################################################################################################
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ingress-external
+  namespace: some-namespace
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/actions.redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP":80,"HTTPS": 443}]'
+    alb.ingress.kubernetes.io/certificate-arn: acm:certificate:arn
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/subnets: 123456789,95481321,321457987
+    alb.ingress.kubernetes.io/security-groups: sg-23157498
```

Comment:
`alb.ingress.kubernetes.io/security-groups`, `alb.ingress.kubernetes.io/security-subnets ` `alb.ingress.kubernetes.io/ssl-policy` should be removed. These three all have sensible default&auto-discover functionality.
Our goal is to make number annotations be as less as possible.

Reply:
I added them because it wasn't intuitive that they were auto discoverable when we first walking through the examples.  I can add some commenting around these to show they are optional and should be auto discovered if you've taken the appropriate steps.

Reply:
What if they were included but commented out?

Reply:
Done.